src/stores/search/search-params-slice.ts

```diff
@@ -26,12 +26,13 @@ export const createSearchParamsSlice: StateCreator<
     indexName: "",
     fragmentSize: 100,
     from: 0,
-    size: 10,
+    // TODO: revert to 10
+    size: 3,
```

Comment:
revert to 10

Reply:
Fixed in [37fde76](https://github.com/knaw-huc/textannoviz/pull/133/commits/37fde76645c9a07d2ec144f1e293c1776b5b99f9)